docs/standard/serialization/binaryformatter-migration-guide/choose-a-serializer.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "BinaryFormatter migration guide: Choose a serializer"
+description: "Compare the capabilities and trade-offs of serializers to choose a replacement for BinaryFormatter."
+ms.date: 5/31/2024
+no-loc: [BinaryFormatter, Serialization]
+helpviewer_keywords:
+  - "BinaryFormatter"
+  - "serializing objects"
+  - "serialization"
+  - "objects, serializing"
+---
+
+# Choose a serializer
```

Comment:
@gewarren this feels odd to me; I'd say "Choosing a serializer" or "How to use a serialzier" but I guess active voice like this is the encouraged style?

Reply:
Acrolinx and [other title guidance](https://review.learn.microsoft.com/en-us/help/platform/metadata-required-articles?branch=main#title) trains me not to use -ing words. I agree it might not always sound the best.